src/Blueprint/Blueprint/help/Get-AzBlueprintArtifact.md

```diff
@@ -0,0 +1,340 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Blueprint.dll-Help.xml
+Module Name: Az.Blueprint
+online version: https://docs.microsoft.com/en-us/powershell/module/az.blueprint/get-azblueprintartifact
+schema: 2.0.0
+---
+
+# Get-AzBlueprintArtifact
+
+## SYNOPSIS
+Get artifact from a blueprint.
+
+## SYNTAX
+
+```
+Get-AzBlueprintArtifact [-Name <String>] -Blueprint <PSBlueprintBase> [-BlueprintVersion <String>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzBlueprintArtifact -Name MyArtifact -Blueprint $bp -BlueprintVersion 1.0
```

Comment:
@filizt please add the corresponding output that the user would see when running this cmdlet

This comment applies to all new examples in the markdown files

Reply:
@filizt are you able to make this change?